modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -360,12 +360,6 @@ TensorMechanicsAction::act()
       auto action = MooseSharedNamespace::static_pointer_cast<MooseObjectAction>(
           _action_factory.create(type, name() + "_gps", action_params));
       _awh.addActionBlock(action);
-      if (isParamValid("extra_vector_tags"))
```

Comment:
These are covered by the `action_params.applyParameters` line above.

Reply:
Whoops, I guess these were my fault.  Thanks for catching that! I think there is a test that would fail if these weren't being passed through, right?